Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.fedoraproject.org/fedora:latest
```

Comment:
If you're using Podman, I'd suggest naming this file `Containerfile` and not `Dockerfile`

Reply:
I did not know this is an option. Will do!